src/errors.js

```diff
@@ -95,3 +95,16 @@ export function onlyErrorsWithCode(
     return errorHandler(error);
   };
 }
+
+export function isErrorWithCode(
```

Comment:
I guess we have to wait until the tests are finished to delete `onlyErrorsWithCode`?